core/dbt/include/global_project/macros/adapters/apply_grants.sql

```diff
@@ -0,0 +1,70 @@
+{% macro get_show_grant_sql(relation) %}
+{{ return(adapter.dispatch("get_show_grant_sql", "dbt")(relation)) }}
+{% endmacro %}
+
+{% macro default__get_show_grant_sql(relation) %}
+show grants on {{ relation.type }} {{ relation }}
+{% endmacro %}
+
+{% macro get_grant_sql(relation, grant_config) %}
+{{ return(adapter.dispatch('get_grant_sql', 'dbt')(relation, grant_config)) }}
+{% endmacro %}
+
+{% macro default__get_grant_sql(relation, grant_config) %}
```

Comment:
**Nit**

For `default__get_grant_sql` + `default__get_revoke_sql`: Let's switch out most of the `{% %}` for `{%- -%}`, to avoid tons of whitespace in the logs. This:
```
On model.testy.view: /* {"app": "dbt", "dbt_version": "1.2.0b1", "profile_name": "garage-postgres", "target_name": "dev", "node_id": "model.testy.view"} */

                    
    

                    
    
        
        
            
                grant select on table "jerco"."dbt_jcohen"."view" to other;
            
                grant select on view "jerco"."dbt_jcohen"."view" to another;
            
        
    

                
```
Should become (at least a little better):
```
/* {"app": "dbt", "dbt_version": "1.2.0b1", "profile_name": "garage-postgres", "target_name": "dev", "node_id": "model.testy.table"} */

                    
                    
                grant select on table "jerco"."dbt_jcohen"."table" to other;
            
                grant select on table "jerco"."dbt_jcohen"."table" to another;
            
                
```